proxmoxtf/resource/file.go

```diff
@@ -95,6 +96,15 @@ func File() *schema.Resource {
 				Description: "The file name",
 				Computed:    true,
 			},
+			mkResourceVirtualEnvironmentFileFileMode: {
+				Type: schema.TypeString,
+				Description: `The file mode in octal format, e.g. "0700" or "600".` +
+					`Note that the prefix "0o" or "0x" is not supported!`,
+				Optional:         true,
+				ValidateDiagFunc: validators.FileMode(),
+				// Allow to update only the file mode even when the content has not changed.
+				ForceNew: true,
```

Comment:
Hm... I think the comment should be updated or removed. `ForceNew` is kinda opposite to what the comment says.

I vaguely remember there was some reason behind having all attributes with `ForceNew: true`, tho can't recall much details.

Reply:
I added the comment to make clear that this is necessary because the file mode won't be updated for existing files, at least not when the content did not change too. I guess the reason for the heavy use of `ForceNew` is exactly this so the comment is superfluous anyway.